content/docs/examples/multicluster/split-horizon-eds/index.md

```diff
@@ -0,0 +1,377 @@
+---
+title: Multicluster with Split Horizon EDS
+description: Example of leveraging Istio's Split Horizon EDS to create a Multi-Cluster.
+weight: 85
+keywords: [kubernetes,multicluster]
+---
+
+This example demonstrates how to use a variation of the [Istio multicluster](/docs/setup/kubernetes/multicluster-install/vpn/) together with _Split Horizon EDS_ (introduced in Istio 1.1) to join two Kubernetes clusters together to allow access to remote instances of a service via the remote Istio ingress gateway. By following the steps in this example you will setup a topology of two clusters as described in the following diagram:
+
+  {{< image width="80%" ratio="36.01%"
+  link="./diagram.svg"
+  caption="Istio mesh spanning multiple Kubernetes clusters with the Split Horizon EDS configured"
+  >}}
+
+The `local` cluster hosts the Istio Pilot and other Istio components while the `remote` cluster only runs the Istio Citadel, Sidecar Injector and Ingress gateway.
+However, unlike the requirements of [Istio multicluster](/docs/setup/kubernetes/multicluster-install/vpn/), no VPN connectivity is required nor direct network access between workload from different clusters.
+
+## Before you begin
+
+In addition to the prerequisites for installing Istio the following setup is required for this example:
+
+* Two Kubernetes clusters with Load Balancers configured (also known as `local` and `remote`)
+
+* Access to the Kubernetes API server of remote cluster from the local
+
+Throughout this tutorial we will assume that `kubectl` can access local and remote clusters with the `--context` flag.
+Setup the following two environment variables with the context names of your configuration:
+
+{{< text bash >}}
+$ export CTX_LOCAL=<KUBECONFIG_LOCAL_CONTEXT>
+$ export CTX_REMOTE=<KUBECONFIG_REMOTE_CONTEXT>
+{{< /text >}}
+
+## Example multicluster setup
+
+This example procedure installs Istio with mutual TLS enabled for the control plane and application pods. We create the `cacerts` secret on both local and remote by using the same Istio certificate from samples as the shared root CA.
+
+The steps below also set up a remote cluster with a selector-less service and an endpoint for `istio-pilot.istio-system` that has the address of the local Istio ingress gateway. To allow that we are securely exposing the local pilot to remote cluster(s) through the ingress gateway without terminating the mTLS.
```

Comment:
To allow that.... -- this reads odd.   I think you meant something like This enables us to securely expose the local pilot to remote cluster(s)....